src/main/java/org/prebid/cache/handlers/PostCacheHandler.java

```diff
@@ -129,5 +144,28 @@ private long adjustExpiry(Long expiry) {
             return expiry;
         }
     }
+
+    private void sendRequestToSecondaryPrebidCacheHosts(RequestObject requestObject, String secondaryCache) {
+        if(!secondaryCache.equals("yes")) {
+            config.getSecondaryIps().forEach(ip -> {
+                WebClient.create().post().uri(uriBuilder -> uriBuilder.scheme(config.getSecondaryCacheScheme())
+                        .host(ip).port(config.getSecondaryCachePort()).path(CACHE_PATH)
```

Comment:
It's dangerous to hard-code "/cache" into the path as cache application might be exposed via reverse-proxy. It's better to move that part into configuration.